lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -653,6 +655,33 @@ OpFoldResult AtenEqIntOp::fold(ArrayRef<Attribute> operands) {
                               [](int64_t a, int64_t b) { return a == b; });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenEqFloatOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenEqFloatOp::fold(ArrayRef<Attribute> operands) {
```

Comment:
all folders need unit tests in https://github.com/llvm/mlir-npcomp/blob/d6b6c0268cfd6ead0aa5f2f21ba02bbe8ead3180/test/Dialect/Torch/canonicalize.mlir